apps/ai-backoffice/lib/workers/pipeline.ts

```
@@ -0,0 +1,126 @@
+export type OutgoingEventData =
+  | ProgressEventData
+  | ReadyEventData
+  | ResultEventData
+
+class PipelineSingleton {
+  static instance?: Pipeline
+
+  static async init(...args: Parameters<typeof pipeline>) {
+    this.instance = await pipeline(...args)
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)